File: frontend/StudentManagement.Shared/Dtos/Diem/DiemResponse.cs
using System;

namespace StudentManagement.Shared.Dtos.Diem;

public class DiemResponse
{
    public string IdHocSinh { get; set; } = string.Empty;
    public string TenHocSinh { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
    public decimal? Diem15 { get; set; }
    public decimal? Diem45 { get; set; }
    public decimal? DiemCk { get; set; }
    public decimal? DiemTb { get; set; }
}

File: backend/src/main/java/com/demo/student_management/dto/diem/DiemResponse.java
package com.demo.student_management.dto.diem;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class DiemResponse {

    private String idHocSinh;

    private String tenHocSinh;

    private String idMonHoc;

    private String idHocKy;

    private BigDecimal diem15;

    private BigDecimal diem45;

    private BigDecimal diemCk;

    private BigDecimal diemTb;
}